pkgs/applications/networking/browsers/firefox/default.nix

```diff
@@ -99,6 +99,7 @@ stdenv.mkDerivation rec {
     '' + lib.optionalString enableGTK3
     ''
       wrapProgram "$out/bin/firefox" \
+        --argv0 "$out/bin/firefox" \  # argv[0] must point to firefox itself
```

Comment:
And even when I fix it, this wrapper doesn't work for me. It seems like the `.firefox-wrapped` basename is important, after all.